lib/ig.js

```diff
@@ -184,10 +197,30 @@ ${match[1]}
   };
   fs.copySync(patchPath, sdPath, { filter: filterFunc });
 
-  const hideSupporting = config.implementationGuide && config.implementationGuide.primarySelectionStrategy
+  const hideSupporting = config.implementationGuide
+  && config.implementationGuide.primarySelectionStrategy
   && config.implementationGuide.primarySelectionStrategy.hideSupporting;
 
+  const usingNamespaceStrategy = config.implementationGuide
+  && config.implementationGuide.primarySelectionStrategy
+  && (config.implementationGuide.primarySelectionStrategy.strategy === 'namespace');
+
   for (const extension of fhirResults.extensions.sort(byName)) {
+    if (primaryExtensionUrls.has(extension.url)) {
+      for (const element of extension.snapshot.element) {
+        if (element.binding
+        && element.binding.valueSetReference
+        && element.binding.valueSetReference.reference) {
+          primaryValueSetUrls.add(element.binding.valueSetReference.reference);
+        }
+  
+        if (element.path.endsWith('.system')
+        && element.fixedUri) {
+          primaryCodeSystemUrls.add(element.fixedUri);
+        }
```

Comment:
The above 10 lines or so could probably be DRYer, in that they are the same as lines 154 - 163.  But... fine for now given the looming deadline.